src/workloads/execution/SSBColumnStoreIndex.yml

```diff
@@ -0,0 +1,595 @@
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_1Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q4_2Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_2Query
+    verbosity:
+      executionStats
+- OperationMetricsName: Q4_3Query
+  OperationName: RunCommand
+  OperationCommand:
+    explain: *Q4_3Query
+    verbosity:
+      executionStats
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 1
+
+SetupPhase: &SetupPhase 0
+BuildColumnStoreIndexPhase: &BuildColumnStoreIndexPhase 1
+ColumnstoreScanPhase: &ColumnstoreScanPhase 2
+DropColumnstoreIndexPhase: &DropColumnstoreIndexPhase 3
+CollScanPhase: &CollScanPhase 4
+MaxPhases: &MaxPhases 5
+
+ColumnStoreIndexName: &ColumnStoreIndexName $**_columnstore
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*SetupPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Thread: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            internalQueryFrameworkControl: "trySbeEngine"
+
+- Name: BuildColumnStoreIndex
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*BuildColumnStoreIndexPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Thread: 1
+        Database: *db
+        Operations:
+        - OperationMetricsName: BulkBuildColumnStoreIndex
+          OperationName: RunCommand
+          OperationCommand:
+            createIndexes: *Collection
+            indexes:
+            - key: {"$**": "columnstore"}
+              name: *ColumnStoreIndexName
+
+# The 'ColumnstoreScan' actor measures the perf with columnar indexes.
+- Name: ColumnstoreScan
+  Type: RunCommand
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*ColumnstoreScanPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Operations: *TestOperations
+
+# The 'DropColumnstoreIndexes' actor drops columnstore indexes.
+- Name: DropColumnstoreIndex
+  Type: RunCommand
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*DropColumnstoreIndexPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+        Database: *db
+        Operations:
+        - OperationName: RunCommand
+          OperationCommand:
+            dropIndexes: *Collection
+            index: *ColumnStoreIndexName
+
+# The 'CollScan' actor measures the perf without columnar indexes.
+- Name: CollScan
+  Type: RunCommand
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [*CollScanPhase]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *TestRepeatCount
+        Database: *db
+        Operations: *TestOperations
```

Comment:
If there was rowstore fallback when running with CSI this might "warm up" the data for the collscan. Again, I have no idea whether that actually happens or matters.

Reply:
That's definitely true; some of these queries aren't able to use CSI yet. My as above, I think the size of the dataset make this inconsequential.

Reply:
while I tend to agree with you that the size of the dataset _should_ make this inconsequential, with performance our intuitions are often wrong... is there any way to confirm our expectations? I also asked on [on Slack](https://mongodb.slack.com/archives/C01VD0LQZED/p1668536753264619) whether we could observe actual disk activity for genny workloads.